<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genius Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/images/settings16.png</normaloff>:/images/images/settings16.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="General_Tab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>341</width>
           <height>151</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="VL2">
          <item>
           <widget class="QCheckBox" name="openMinimizedCheckbox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to start genius minimized and run in background&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Open Minimized</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pasteAutomaticalyButoon">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to paste selected item automaticaly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Paste Automaticaly</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="saveImages">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to save images to the history&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Save Images</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="saveURLs">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to save URLs to the history&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Save URLs</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="HL3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;the maximum number of items store in the history. maximum is 100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Maximum Items In History</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maximumItemsInHistory">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="View_Tab">
        <attribute name="title">
         <string>View</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>294</width>
           <height>59</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="VL3">
          <item>
           <widget class="QCheckBox" name="ShowInSingleLineCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;if this is checked all text items in the&lt;span style=&quot; font-weight:600;&quot;&gt; history manager and menus will show in single line&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Show In Single Line</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="HL2">
            <item>
             <widget class="QCheckBox" name="LimitTextLengthCheckbox">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;limit charactor length of items &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Limit View Item TextLength to</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="TextLimitSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="ShortcutsTab">
        <attribute name="title">
         <string>Shortcuts</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>394</width>
           <height>241</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="OpenSelectorCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;OpenSelector Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, Selector will popup and you can select item to paste from clipboard history&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Open Selector</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="openSelector">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Open Selector&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="clearHistoryCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;ClearHistory Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, clipboard history will clear. and a notification will popup containing number of items deleted&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Clear History   </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="clearHistory">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Clear History&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="pasteLastCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;PasteLast Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, the last item in the history will be copied to the current window. last item will be the new top item. (0th index of the history)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Paste Last        </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="pasteLast">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Paste Last&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="openManagerCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;ShowManager Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, The Manager Window (main window that help to manage the history) will show.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Manager           </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="openManager">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Show Manager Window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="openSettingsCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;OpenSettings Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, this window (settings window) will open&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Settings            </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="openSettings">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Show Settings Window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="directCopyCB">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;check if you want to enable &lt;span style=&quot; font-weight:600;&quot;&gt;DirectCopy Global Hotkey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;if you press this hotkey, the copy command executes and send data directly to the genius.(without saving in clipboard)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Direct Copy     </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="directCopy">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hotkey for Direct copy&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ff0000;&quot;&gt;*genius should restart to change hotkeys&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HL1">
       <item>
        <widget class="QPushButton" name="defaultButton">
         <property name="text">
          <string>Default</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/images/images/reset16.png</normaloff>:/images/images/reset16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/images/images/ok16.png</normaloff>:/images/images/ok16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/images/images/cancel16.png</normaloff>:/images/images/cancel16.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>cancelButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
